# before the config gets created the window is too small
# https://bugs.launchpad.net/getdeb.net/+bug/1193694
Index: aseprite-0.9.5/src/modules/gui.cpp
===================================================================
--- aseprite-0.9.5.orig/src/modules/gui.cpp	2012-08-25 00:16:26.000000000 +0200
+++ aseprite-0.9.5/src/modules/gui.cpp	2013-06-24 21:34:14.949910049 +0200
@@ -239,8 +239,8 @@
 
 static void load_gui_config(int& w, int& h, bool& maximized)
 {
-  w = get_config_int("GfxMode", "Width", 0);
-  h = get_config_int("GfxMode", "Height", 0);
+  w = get_config_int("GfxMode", "Width", 800);
+  h = get_config_int("GfxMode", "Height", 600);
   screen_scaling = get_config_int("GfxMode", "ScreenScale", 2);
   screen_scaling = MID(1, screen_scaling, 4);
   maximized = get_config_bool("GfxMode", "Maximized", false);
